ntbs-service/deployments/live.yml

```diff
@@ -98,6 +98,10 @@ spec:
                   key: reporting
             - name: ExternalLinks__ReportingOverview
               value: ""
+            - name: ExternalLinks__ClusterReportBase
+              value: ""
+            - name: ExternalLinks__ClusterReportReplacementSymbol
+              value: ""
```

Comment:
Generally we don't include config that we don't want to set in these files... is there a reason to do so here?

Reply:
Nope there is no reason, removed now. Sorry about that!